cli/src/main.rs

```diff
@@ -8,8 +8,10 @@ mod compiler;
 mod log_id;
 
 fn main() {
-    let log_handler = LogEventHandlerBuilder::new()
-        .write_to_console()
+    let _ = logid::set_filter!("info(infos)");
```

Comment:
May I ask why is this a macro? Looking at the source code I feel like it could (and should) be a function. Also, would it be possible to use `enum`s for filter level? Using string feels very error prone to be honest.

Reply:
The idea was to have an environmental variable to set the filter, but the variable would need to be set during compiletime, and I was not able to get this to work using environmental variables in cargo's config.toml file.
With the string approach, setting the filter is the same for the environmental variable or via macro/function.

I also just wanted to get the filter working to have at least some filter.

The macro could be a function, but this would be a change in logid, and this might change in the future if I add another way to set the filter without using the string syntax.

Reply:
changed to newest logid version that offers a filter builder instead of the `set_filter` macro.

filter builder was implemented by @nfejzic. Thank you :)